rofl.yaml

```diff
@@ -35,8 +35,8 @@ deployments:
           min_tcb_evaluation_data_number: 18
           tdx: {}
       enclaves:
-        - XoGWlUr9yeXME/6nPHIlASaS0/q4LZ2vExFbUoWrF9sAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==
-        - 69jynVbbjXkNgoalE83L47POjbOMJ0yOcd+LrUkxOiEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==
+        - q5o8rBEzvhk9fWsHFha68CAJaMUaGYY87TTg5sD4yQEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==
```

Comment:
Did you compile the bundle with the new Oasis CLI 0.17.0? It brings support for reproducible builds on all platforms, but breaks existing enclave IDs.